pkg/karmadactl/cmdinit/kubernetes/deployments.go

```diff
@@ -108,7 +108,7 @@ func (i *CommandInitOption) makeKarmadaAPIServerDeployment() *appsv1.Deployment
 	}
 
 	// Probes
-	livenesProbe := &corev1.Probe{
+	livenessProbe := &corev1.Probe{
```

Comment:
Was this modification discovered by `misspell`? In fact, I have some doubts that there should be more than that spelling errors in our project, but why didn't they find them?

Reply:
> Was this modification discovered by `misspell`? In fact, I have some doubts that there should be more than that spelling errors in our project, but why didn't they find them?

NO I found livenesProbe at first and try to use misspell to discover it and the result is `memeber` error

> Hi @wuyingjun-lucky, thanks for your contribution!
> 
> I think you can put `gci` and `misspell` into two pr, let's deal with them one by one. For `gci`, it looks good to me.

OK